frontend/Login/AccountType.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { Text, View, StyleSheet, Pressable } from "react-native";
+import { Feather, FontAwesome5 } from "@expo/vector-icons";
+import { NativeStackScreenProps } from "@react-navigation/native-stack";
+import LargeOption from "./LargeOption";
+import Header from "../components/Header";
+import { RootStackParamList } from "../route-settings";
+
+const styles = StyleSheet.create({
+  buttonWrapper: {
+    display: "flex",
+    flexDirection: "row",
+  },
+  description: {
+    paddingLeft: 40,
+    paddingTop: 100,
+    fontSize: 20,
+  },
+});
+
+type AccountTypeProps = NativeStackScreenProps<
```

Comment:
Would you be okay with moving this to ./route-settings.tsx and exporting it from there? Some of the other navigation props are in there so it would just help with continuity

Reply:
I don't think this would be a good idea ... if anything we should be pulling those props out and leaving them with the component. I'd rather not have a file of 50+ prop types at the end of this project lol

Reply:
Good point actually

Reply:
I agree with Fin actually... I think it makes more sense to couple a UI page w/ a simple prop import rather than couple it with big routing elements like NativeStackScreenProps and RootStackParamList. That way someone who doesn't know what's going on with routing can still understand what is going on in the ui file. Also, this router is only for the four main pages plus a couple other things, so I don't forsee there being 50+ prop type exports.